relays/messages-relay/src/message_lane_loop.rs

```diff
@@ -71,10 +71,21 @@ pub struct MessageDeliveryParams {
 	pub max_messages_in_single_batch: MessageNonce,
 	/// Maximal cumulative dispatch weight of relayed messages in single delivery transaction.
 	pub max_messages_weight_in_single_batch: Weight,
+	/// Maximal cumulative size of relayed messages in single delivery transaction.
+	pub max_messages_size_in_single_batch: usize,
+}
+
+/// Message weights.
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub struct MessageWeights {
+	/// Message dispatch weight.
+	pub weight: Weight,
+	/// Message size.
```

Comment:
```suggestion
	/// Message size - number of bytes in encoded payload.
```

Reply:
I'm not sure - what do you mean here? It is actually size of value stored in the messages storage map (the value that is proved later).

Reply:
Ok, looks like I misinterpreted dash as a minus sign :) I thought you're subtracting encoded payload size from the message size

Reply:
Hah, yeah I just meant to clarify what "size" is. Feel free to re-phrase to make it clearer, maybe a comma would less ambiguous.